object_manager.go

```diff
@@ -40,7 +43,7 @@ type IBObjectManager interface {
 	GetNetworkContainer(netview string, cidr string) (*NetworkContainer, error)
 	GetNetworkView(name string) (*NetworkView, error)
 	GetPTRRecordByRef(ref string) (*RecordPTR, error)
-	GetZoneAuthByRef(ref string) (*ZoneAuth, error)
+	GetZoneAuthByRef(ref string) (ZoneAuth, error)
```

Comment:
We recommend to pass a pointer to an object rather than the object. Is there any specific reason for this change ?

Reply:
This is how the ObjectManager was coded, I'm only updating the interface to adhere to that. I agree that it should be a pointer.